sql/mob_spawn_points.sql

```diff
@@ -63027,37 +63027,37 @@ INSERT INTO `mob_spawn_points` VALUES (17510679,'Aldo','Aldo',21,-0.021,-197.555
 -- ------------------------------------------------------------
 
 -- Ark Angels 1 (ZM14)
-INSERT INTO `mob_spawn_points` VALUES (17514497,'Ark_Angel_HM','Ark Angel HM',1,-13.833,-18.716,13.796,44);
```

Comment:
Lets remove the pos changes from this MR, I already fixed these via a direct MR to LSB